examples/operator/README.md

```diff
@@ -9,47 +9,47 @@ kubectl apply -f operator.yaml
 
 # Bring up initial cluster and commerce keyspace
 kubectl apply -f 101_initial_cluster.yaml
-vtctlclient ApplySchema -sql="$(cat create_commerce_schema.sql)" commerce
-vtctlclient ApplyVSchema -vschema="$(cat vschema_commerce_initial.json)" commerce
+kvtctl ApplySchema -sql="$(cat create_commerce_schema.sql)" commerce
```

Comment:
@morgo wanted to use the original names. The intent was that we'll have a config directory like kubectl does. But we haven't done that part yet.

But I'm ok with using a different name until that part is done. This has confused me also a couple of times.